README.md

```diff
@@ -1,23 +1,86 @@
 # @pagopa/io-react-native-crypto
 
-Module to generate and sign with crypto keys on react-native platform
+Module to generate and sign with crypto keys backed on device security hardware on react-native platform.
 
 ## Installation
 
 ```sh
-npm install @pagopa/io-react-native-crypto
+yarn add @pagopa/io-react-native-crypto
```

Comment:
Should we show how to directly add the module from GitHub this way `yarn add https://github.com/pagopa/io-react-native-crypto` or  `yarn add pagopa/io-react-native-crypto` until the module is not published on the npm registry?

Reply:
I would leave it like that and start working on npm distribution as soon as possible after this iteration 😄